### PR TITLE
fix: pdf for collection reports

### PR DESCRIPTION
**Pull request by AmitojSingh1**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by kunalfleethawks — APPROVED** [review]

File: src/app/pages/reports/accounts/customer-collection/customer-collection.component.ts [hunk 1]
```diff
@@ -198,16 +200,14 @@ setCustomer(cName){
   async generatePDF() {
     let data = document.getElementById("print_wrap");
     html2pdf(data, {
-      margin: [0.5, 0.5,0.5,0.5],
-      pagebreak: { mode: ["avoid-all"] },
+      margin: 0,
```
Comment (kunalfleethawks) [inline src/app/pages/reports/accounts/customer-collection/customer-collection.component.ts:203]: Why did you revert the code ?

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please respond to review comment

File: src/app/pages/reports/accounts/customer-collection/customer-collection.component.ts (hunk 1 above)
Comment (AmitojSingh1) [reply src/app/pages/reports/accounts/customer-collection/customer-collection.component.ts:203]: this page break is not working as expected. and adding margin and shrinking the table to the right. so I have to remove it from here. but for longer and combined reports it's working fine. so m using there.

Comment (kunalfleethawks) [reply src/app/pages/reports/accounts/customer-collection/customer-collection.component.ts:203]: Ok. I hope it does not chop off the text in the long run and we have to do it again.

**Pull request merged**